docs/dev/build.md

```diff
@@ -344,56 +401,217 @@ There are two options how to use OpenVINO on Apple Silicon:
+% cmake --build . --config Release --parallel $(sysctl -n hw.ncpu)
 ```
 All built binaries are located in `<openvino_source_dir>/bin/<arm64 | intel64>/Release/` and wheel packages are located in `<openvino_build_dir>/wheels`. 
 
 5. (Optional install) Once you have built OpenVINO, you can install artifacts to a preferred location:
 ```sh
-cmake -DCMAKE_INSTALL_PREFIX=<installation location> -P cmake_install.cmake
+% cmake -DCMAKE_INSTALL_PREFIX=<installation location> -P cmake_install.cmake
 ```
 
 ### Building x86_64 binaries
 
-Since OSX version 11.x and Xcode version 12.2, the Apple development tools allows to compile arm64 code on x86 hosts and vice-versa. Based on this, OpenVINO can be compiled as x86_64 binary, then run on Apple Silicon hosts using [Rosetta]. For this, first of all Rosetta must be installed:
+Since OSX version 11.x and Xcode version 12.2, the Apple development tools allow to compile arm64 code on x86 hosts and vice-versa. Based on this, OpenVINO can be compiled as x86_64 binary, then run on Apple Silicon hosts using [Rosetta]. To do this, you must first install Rosetta:
 
 ```sh
-softwareupdate --install-rosetta
+% softwareupdate --install-rosetta
 ```
 
 Then try to compile OpenVINO using the steps above, but adding `-DCMAKE_OSX_ARCHITECTURES=x86_64` on cmake configure stage. But, **don't enable any system library usage explicitly** via CMake options, because they have `arm64` architecture, e.g.:
 ```sh
-file /opt/homebrew/Cellar/tbb/2021.5.0_2/lib/libtbb.12.5.dylib
+% file /opt/homebrew/Cellar/tbb/2021.5.0_2/lib/libtbb.12.5.dylib
 /opt/homebrew/Cellar/tbb/2021.5.0_2/lib/libtbb.12.5.dylib: Mach-O 64-bit dynamically linked shared library arm64
 ```
 
-> **Note:** since you are building with `universal2` python libraries, wheel package is created with name `openvino-2022.3.0-000-cp39-cp39-macosx_12_0_universal2.whl` and have proper `universal2` tags, so can *potentially* be used on both Apple Silicon and Intel CPU.
+The same goes for other external dependencies like `libusb`. If you want to enable extra functionality like enable MYRIAD plugin build, you need to provide either x86_64 or universal2 `libusb` library. All other steps are the same as for usual compilation: build, install.
+
+> **NOTE**: When building with `universal2` python libraries, wheel package is created with the `openvino-2022.3.0-000-cp39-cp39-macosx_12_0_universal2.whl` name and have a proper `universal2` tags, so they can *potentially* be used on both Apple Silicon and Intel CPU.
+
+
+<p>
+</details>
+
 
+<p>
+</details>
 
+<details><summary>Android</summary>
 <p>
+
+1. Download and unpack [Android NDK](https://developer.android.com/ndk/downloads). Let's assume that `~/Downloads` is used as a working folder.
+  ```sh
+  cd ~/Downloads
+  wget https://dl.google.com/android/repository/android-ndk-r20-linux-x86_64.zip
+
+  unzip android-ndk-r20-linux-x86_64.zip
+  mv android-ndk-r20 android-ndk
+  ```
+
+2. Clone submodules
+  ```sh
+  cd openvino
```

Comment:
We already cloned repositories and submodules

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/16063/commits/562c6c32912eb936d9ca384c7caa9cf772ce2fa2